account-kit/infra/src/alchemyTransport.ts

```diff
@@ -176,6 +179,7 @@ export function alchemy(config: AlchemyTransportConfig): AlchemyTransport {
           }),
         });
       }
+      mutateRemoveTrackingHeaders(fetchOptions);
```

Comment:
why do we remove from here? the transport below is used to make alchemy calls, so it's fine to have the headers there (unless this is doing a reset)

Reply:
the only place that must have them removed is on line 178, so you can probably do it without the mutation and just have the `mutateRemoveTrackingHeaders` function sanitize the headers in the `config.fetchOptions`

Reply:
Should be doing just the config.fetchOptions nows, that makes a lot of sense.